### PR TITLE
DELL3437 id:88 for xiaoleGun(zoran)

### PR DESCRIPTION
**Pull request by xiaoleGun**

### CODE REVIEW

**Comment by vandroiy2013** [comment]
You broke XML structure in Resources/PinConfigs.kext/Contents/Info.plist. 
Build project before making pull request. 

Also is this yours resources? If no you have to approve this changes with author or you RP will be closed.

**Comment by xiaoleGun** [comment]
> 您在资源/PinConfigs.kext/内容/Info.plist 中打破了 XML 结构。
> 在发出拉取请求之前生成项目。
> 
> 这也是你的资源吗？如果没有，您必须批准此更改与作者或 RP 将关闭。

I'm sorry about that, I will patche it up and pull it

**Comment by vandroiy2013** [comment]
One more time! If this resources is not yours, and i believe they not because you are a first-time contributor, you must approve this changes with author! Otherwise PR will be closed.

**Comment by Andrey1970AppleLife** [comment]
CC @daliansky

**Comment by xiaoleGun** [comment]
> One more time! If this resources is not yours, and i believe they not because you are a first-time contributor, you must approve this changes with author! Otherwise PR will be closed.



> One more time! If this resources is not yours, and i believe they not because you are a first-time contributor, you must approve this changes with author! Otherwise PR will be closed.

请问，我刚刚提交的可以吗？我声明一下，这是我添加的ID，上次让daliansky帮忙提交的，因为我13岁，不大会用Git。
Excuse me, can I just submit it? I want to make a statement. This is the ID I added. Last time I asked daliansky to help me submit it, because I'm 13 years old and don't use git very much.

**Comment by vandroiy2013** [comment]
> Excuse me, can I just submit it? I want to make a statement. This is the ID I added. Last time I asked daliansky to help me submit it, because I'm 13 years old and don't use git very much.

Let's wait @daliansky reply. 
If i understand correctly that was you the author of this layout-id 88 and you just asked @daliansky to push it here. Am i correct?

**Comment by xiaoleGun** [comment]
> > 对不起，我能提交吗？我想发表声明。这是我添加的 ID。上次我请Daliansky帮我提交，因为我13岁，不用git很多。
> 
> 让我们等待@daliansky回复。
> 如果我明白，这是你这个布局id 88的作者，你只是要求@daliansky把它推到这里。我是对的吗？

是的，我英语不好，请谅解，谢谢
Yes, my English is not good, please understand, thank you

**Comment by vandroiy2013** [comment]
> Yes, my English is not good, please understand, thank you

OK. Thanks. Will be merged in next release.

**Comment by Andrey1970AppleLife** [comment]
You broke formatting in Resources/PinConfigs.kext/Contents/Info.plist.
Build a kext in Xcode for fix.

**Comment by vandroiy2013** [comment]
@Andrey1970AppleLife He already did it
https://github.com/acidanthera/AppleALC/pull/549/commits/f253d2367f9e9c495dfa74ec86e8f23c6cab7ff1

File: Resources/ALC283/layout88.xml [hunk 1]
```diff
@@ -50,66 +50,7 @@
...
-								ucLzD7rCuQy7
-								wjs8vsKfJsLC
-								xgrFwqVeyMIT
-								6srCU+nMwv+M
-								0MKaftbCpcjd
-								wm+p5sL/CfHC
-								HcT8wrp3A8Pi
-								JAzD
-								</data>
+								<data>UNiAwmgCb8KcoIHCjGeGwko3g8KLS3nClQV9wnr7hMLRp43CAIiSwlajlcLnK5jCHMuYwrl9lcJXm4/CBhmQwuJvlMKbxJTC7qyUwtjDl8KU+ZzCnCaewsmuncK/i5/CJtSewp8um8Lex5rC2QCdwq1lnsJ105zC9qmcwrMcnsLj/qDCZs2hwhtCocIFoZ/CCxOewrxwncIKiZrCyO+YwsmtmsID5p7CeNqfwhNMncKWIp3ChrycwiJcm8IkjJ3CsKqfwuByn8Iu8ZvCvyiWwhbdkcIEu4vCUfWLwoYrlcIy5J3Ci9ycwp0lnsK0QaDCOiygwnxdn8IztJ7Ciq2fwhFzosLIZaPCwUOjwo6TosIkR6LC6vehwtrwosIdtJ/CXLmbwlSZmcKDhJXCDFGRwnV6j8JTjY/CrqGQwgqYk8INzpjCuTufwrjlocKviKPC5YqlwgdmpcKZ2aXCGiumwq95osJOIJ/Cxl+ewtWGl8KmPJPC+sSawkdHo8JWB6LCskyhwqk7pcIth6nCh4Wswk+crcK9J6zCYJWqwmVJq8K8063C+1GuwiQAqMJjvZ/Cjo2hwsParsLRXrXCec63wr6IucLzD7rCuQy7wjs8vsKfJsLCxgrFwqVeyMIT6srCU+nMwv+M0MKaftbCpcjdwm+p5sL/CfHCHcT8wrp3A8PiJAzD</data>
```
Comment (vit9696) [inline Resources/ALC283/layout88.xml:53]: Does not look so to me.

Comment (xiaoleGun) [reply Resources/ALC283/layout88.xml:53]: 所以，我重新修改并提交的还是错误的格式吗？
So, am I modifying and submitting the wrong format?

**Comment by Andrey1970AppleLife** [comment]
> @Andrey1970AppleLife He already did it
> [f253d23](https://github.com/acidanthera/AppleALC/commit/f253d2367f9e9c495dfa74ec86e8f23c6cab7ff1)

No. Look diff.

**Comment by vandroiy2013** [comment]
Fixed and merged

**Comment by xiaoleGun** [comment]
> Fixed and merged

谢谢您的合并，非常感谢
Thank you for your merger. Thank you very much

**Pull request merged**